CMakeLists.txt

```diff
@@ -1,4 +1,11 @@
 cmake_minimum_required(VERSION 3.0)
+
+# VERSION_GREATER_EQUAL not available until 3.7
+if ((CMAKE_VERSION VERSION_EQUAL 3.4) OR (CMAKE_VERSION VERSION_GREATER 3.4))
+	# Enable -rdynamic link flag only if ENABLE_EXPORTS is True
+	cmake_policy(SET CMP0065 NEW)
+endif()
+
```

Comment:
Short time to live, but okay, I don't mind.

Reply:
Indeed. It's documentation, not churn.